src/components/tab-link/tab-link.tsx

```diff
@@ -0,0 +1,63 @@
+import Link from 'next/link';
+import style from './tab-link.module.css';
+import {
+  ComponentText,
+  TranslatedString,
+  useTranslation,
+} from '@atb/translations';
+import { Typo } from '../typography';
+import { AnimatePresence, motion } from 'framer-motion';
+
+type TabLinkProps = {
+  activePath: string;
+};
+
+type LinkProps = {
+  title: TranslatedString;
+  href: string;
+};
+
+const links: LinkProps[] = [
+  {
+    title: ComponentText.TabLink.assistant,
+    href: '/' || '/assistant',
```

Comment:
this would evaluate in place, so essentially `href` will hold the value `/` as `'/' || '/assistant'` will say truthy value `/` or truthy value `/assistant` where `'/' == true` and will cause the conditional to return. You can check this in the REPL or in DevTools to see what it returns to.

But this will again cause the startsWith to always be true

Reply:
Ah, I see. I´ll replaced '/' || '/assistant' with the regex expression. By the way, is it okay to leave comments in the code? I think regex expressions often can be a bit difficult to understand, and therefore have a habit of commenting the logic.

Reply:
From commit message

> Removed regex logic from code. This solution seems fine as it sets the correct active path for '/', '/assistant', and 'departures'

I promise you. It doesn't work. Or at least it doesn't work as intended, and if you observe the expected result when testing it, it works by "luck". JavaScript/TypeScript doesn't function this way. 

See what the result of this line is in the console:

<img width="225" alt="Screenshot 2023-12-11 at 16 54 20" src="https://github.com/AtB-AS/planner-web/assets/606374/23d0be4d-4f2f-4ca1-84ec-7c42e111911c">

Without testing it out I think you see it working as isActive will be `true` for both cases, but the underline element with layoutId is reused by framer-animation and only shown once. Since the `/departures` case happens as the last case the underline element will be reused and position under departures-link.

Reply:
I chose to move away from the regex expression as it became more complicated than necessary. Instead, type check was added to ensure that only valid routs can be passed as attributes in <TabLink/>. With this approach, the problem of having to distinguish between if either '/'  or '/assistant' was passed in as a prop is avoided.